### DISCUSSION TITLE
How can apply the migrations up to a specific migration ?

### DISCUSSION THREAD

**Discussion by emyann** [prisma/prisma#7316] [Q&A] [answered]
Hello! I want to write a test validating that any new migration is backward compatible with the previous one and it does not trigger a prompt in production to the user telling that some data could be lost. I'd would like to catch that at dev time so the developer can write the appropriate data migration in the generated SQL file.

To achieve that, I want to apply the migrations until the penultimate one, then apply the last one and assert that there is no prompt due to a potential data loss. 

With the legacy `prisma migrate`, it was possible to apply the migrations up to a specific one with `prisma migrate up 20200223181448 --experimental`, but I can't figure out how to do the same with the latest version of prisma, is it still possible to do that with the CLI ?

**Comment by ryands17** [comment, marked as answer]
@emyann 👋 

Unfortunately we do not support migrations upto a specific point at the moment, but we do have a feature request for the same here where you can add a 👍 so that we can look at the priority :)

https://github.com/prisma/prisma/issues/6267
